File: COM_SIC_Activaciones/SmartWatch_Rest/MessageResponse.vb
'INICIO PROY-140379
Imports System
Imports System.Runtime.Serialization
<Serializable()> _
Public Class MessageResponse

    Private _Header As HeadersResponse
    Private _Body As bodyResponseSmartWatch

    Public Property Header() As HeadersResponse
        Get
            Return _Header
        End Get
        Set(ByVal Value As HeadersResponse)
            _Header = Value
        End Set
    End Property

    Public Property Body() As bodyResponseSmartWatch
        Get
            Return _Body
        End Get
        Set(ByVal Value As bodyResponseSmartWatch)
            _Body = Value
        End Set
    End Property

    Public Sub New()
        _Header = New HeadersResponse
        _Body = New bodyResponseSmartWatch

    End Sub

End Class
'FIN PROY-140379

File: COM_SIC_Entidades/DataPowerRest/HeadersGenerics/MessageResponse.cs
//INICIO PROY-140332 (Try & Buy)
using System;
using System.Runtime.Serialization;

namespace COM_SIC_Entidades.DataPowerRest.HeadersGenerics
{
	/// <summary>
	/// Summary description for MessageResponse.
	/// </summary>
	[Serializable]
	public class MessageResponse
	{
		private HeaderResponse _headerResponse;
		private BodyResponse _bodyResponse;

		public HeaderResponse Header {get{ return _headerResponse;} set{_headerResponse = value;}}
		public BodyResponse Body {get{ return _bodyResponse;} set{_bodyResponse = value;}}

		public MessageResponse()
		{
			_headerResponse = new HeaderResponse();
			_bodyResponse = new BodyResponse();
		}



	}
}
//FIN PROY-140332 (Try & Buy)
